validator/src/commands/contact_info/mod.rs

```diff
@@ -33,7 +33,13 @@ pub fn command(_default_args: &DefaultArgs) -> App<'_, '_> {
 }
 
 pub fn execute(matches: &ArgMatches, ledger_path: &Path) {
-    let contact_info_args = ContactInfoArgs::from_clap_arg_match(matches);
+    let contact_info_args = match ContactInfoArgs::from_clap_arg_match(matches) {
+        Ok(args) => args,
+        Err(e) => {
+            eprintln!("Error parsing contact info args: {e}");
+            exit(1);
+        }
```

Comment:
I know the code currently has a bunch of these, but I'd really like to do away with these. In https://github.com/anza-xyz/agave/pull/5026, I have started work to propagate errors (not finished yet).

I can probably continue working on it tomorrow. This would then simplify things for this PR's and the other ones in the chain

Reply:
yeah, let's get your error handling landing first! 🫡